src/features/groups/list/index.tsx

```diff
@@ -22,35 +28,127 @@ const filterTabs: IFilterTab[] = [
 
 const GroupsList = () => {
   const { t } = useTranslation('groups');
+  const groups = useSelector(groupsListSelector);
+  const dispatch = useDispatch();
 
-  const { data: groups } = useGetTransactionGroupsQuery();
+  const [loadGroups] = useLazyGetTransactionGroupsQuery();
 
   const [activeTab, setActiveTab] = useState<IFilterTab>(filterTabs[0]);
 
+  const [isGroupDetailsMode, setIsGroupDetailsMode] = useState<boolean>(false);
+  const [isAddGroupMembersMode, setIsAddGroupMembersMode] = useState<boolean>(false);
```

Comment:
Why `isAddGroupMembersMode ` flag here, but `GroupMembers` component that depends on this flag you use in another component?
It adds a bit mess, I think.